poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/ZipPackage.java

```diff
@@ -476,32 +489,18 @@ protected void closeImpl() throws IOException {
             IOUtils.closeQuietly(this.zipArchive);
             try {
                 // Copy the new file over the old one if save() succeed
-                if(success) {
-                    FileHelper.copyFile(tempFile, targetFile);
+                if (success) {
```

Comment:
is this the meet and potatoes advantage for swapping from File to Path ?

Reply:
Yeah